packages/web-features/index.ts

```diff
@@ -8,4 +8,5 @@ const { features, groups, snapshots } = JSON.parse(
   readFileSync(jsonPath, { encoding: "utf-8" }),
 ) as WebFeaturesData;
 
+export * from "./types";
```

Comment:
This is surprising to me. Can you explain what this does? When I read the documentation on this, I was under the impression that the `"types"` field in `package.json` was the correct way to provide the types.

Reply:
Indeed, which sets `index.d.ts` as the type declaration file for the package: however, that's automatically generated by typescript from `index.ts`, so if `index.ts` doesn't export a type, `index.d.ts` also doesn't export a type; so in order to be able to access all the types exported in `types.ts`, we have to export everything from that in `index.ts`.

Reply:
Ah ha! OK, thank you—that explains it.